index.html

```diff
@@ -37,7 +37,144 @@ <h3 id="foot">LET’S CONNECT</h3>
       </ul>
     </div>
   </section>
-</div>
-<script src="https://kit.fontawesome.com/3bf83b5817.js" crossorigin="anonymous"></script>
-</body>
-</html>
+    </div>
+      <!--Work section-->
+    <div class="Work section">
```

Comment:
- [ ] Great working on you work section, kindly Add some `padding` between the tags and the button so it aligns with the Figma Template.

Figma Template
![pad-2](https://user-images.githubusercontent.com/48371156/165937663-44cfa731-94dc-41a8-b52c-f9cf8590406d.png)

## Your Design
![pad-1](https://user-images.githubusercontent.com/48371156/165937665-5c2a4fc3-ab1a-41bd-9d3b-ea18b7e6fa5d.png)